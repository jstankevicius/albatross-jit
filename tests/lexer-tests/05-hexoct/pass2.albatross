01234567 0 01 02 000000001
01 012 010203 012340
