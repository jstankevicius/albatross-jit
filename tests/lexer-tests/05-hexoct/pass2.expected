1 1 INT 342391
10 1 INT 0
12 1 INT 1
15 1 INT 2
18 1 INT 1
1 2 INT 1
4 2 INT 10
8 2 INT 4227
15 2 INT 5344
